reviewrot/githubstack.py

```diff
@@ -0,0 +1,166 @@
+        log.debug('Looking for pull requests for %s -> %s/%s ',
+                  'github', uname.login, repo_name)
+        # get list of open pull requests for a given repository
+        pull_requests = repo.get_pulls()
+        if len(list(pull_requests)) == 0:
+            log.debug('No open pull requests found for %s/%s ',
+                      uname.login, repo_name)
+        res_ = []
+        for pr in pull_requests:
+            """
+            find the relative time difference between now and
+            pull request filed to retrieve relative information
+            """
+            rel_diff = relativedelta(datetime.datetime.now(),
+                                     pr.created_at)
+            """
+            find the absolute time difference between now and
+            pull request filed to retrieve absolute information
+            """
+            abs_diff = datetime.datetime.now() - pr.created_at
+            if (state_ is not None and value is not None and
+                    duration is not None):
+                """
+                check if pull request is older/newer than specified time
+                interval
+                """
+                result = self.check_request_state(abs_diff, rel_diff, state_,
+                                                  value, duration)
+                # skip the request if it doesn't match the specified criteria
+                if not result:
+                    log.debug("pull request '%s' is not %s than specified"
+                              " time interval", pr.title, state_)
+                    continue
+            # format the time interval pull request has been filed since
+            time = self.format_duration(rel_diff)
+            # fetch and format comments for pull request
+            comments = []
+            if(pr.comments == 1):
+                comments.append('%s' % ', with 1 comment')
+            elif(pr.comments > 1):
+                comments.append('%s %s %s' % (', with', str(pr.comments),
+                                              'comments'))
+            comments = ' '.join(comments)
+            # format and print the resultant pull request string
+            res = GithubReviewRot(user=pr.user.login,
+                                  title=pr.title,
+                                  url=pr.html_url,
+                                  time=time,
+                                  comments=comments)
+            log.info(res)
+            res_.append(res)
+        return res_
+
+    def check_request_state(self, abs_diff, rel_diff, state_, value, duration):
+        return super(GithubService,
+                     self).check_request_state(abs_diff, rel_diff,  state_,
+                                               value, duration)
+
+    def format_duration(self, rel_diff):
+        return super(GithubService, self).format_duration(rel_diff)
+
+
+class GithubReviewRot(BaseReviewRot):
```

Comment:
Just the file name was 'base' instead of 'basereview'. Hence, the code still works, but it's a good idea to drop the name rot. I can do that